crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -622,8 +625,13 @@ fn handle_positional_only_arguments_separator_comment<'a>(
         return CommentPlacement::Default(comment);
     };
 
-    let is_last_positional_argument = are_same_optional(last_argument_or_default, arguments.posonlyargs.last())
-            // If the preceding node is the default for the last positional argument
+    let is_last_positional_argument =
```

Comment:
- I wonder if we can simplify this now by simply using the argument with default range. Also... did you override the new visit methods in the CommentVisitor?

Reply:
Fixed in https://github.com/astral-sh/ruff/pull/5204/